lib/components/narrative/narrative-itineraries-header.js

```diff
@@ -72,15 +59,31 @@ export default function NarrativeItinerariesHeader ({
         : <>
           <div
             style={{flexGrow: 1}}
-            title={titleText}
+            title={intl.formatMessage({
+              id: 'components.NarrativeItinerariesHeader.titleText',
+              values: {
+                issueNum: errors.length,
+                itineraryNum: itineraries.length,
+                pending: pending ? 'true' : 'false'
+              }
+            })}
```

Comment:
Can `FormattedMessage` be used here instead?

Reply:
From Binh's reply on Teams: some JSX props such as alt, title or placeholder require a string, so we have to get intl. <FormattedMessage> will not work for these props (it returns a React Fragment).